src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
 // write your code here
+const rows = document.querySelector('tr');
```

Comment:
The `document.querySelector('tr')` will only select the first `<tr>` element in the document. To select all rows, use `document.querySelectorAll('tr')` instead, which returns a NodeList of all `<tr>` elements.